src/firefly/js/drawingLayers/HiPSMOC.js

```diff
@@ -405,12 +453,44 @@ function updateMocData(dl, plotId) {
                  startIdx, endIdx, updateStatus.storedSidePoints);  // handle max chunk
              updateStatus.processedTiles.push(...moreObjs);
              if (updateStatus.processedTiles.length >= updateStatus.totalTiles) {
-                 abortUpdate(dl, updateStatusAry, plotId, LayerUpdateMethod.byTrueAry);
+                 completeAsyncUpdate(dl, updateStatusAry, plotId, LayerUpdateMethod.byTrueAry);
              }
          }
      }
 }
 
+function getAutoDrawStyle(plot, color,autoUsesOnlyOutline=false) {
+    const [r,g,b]= toRGBA(color);
+    const fov= getFoV(plot);
+    if (!fov) return {style:Style.FILL, color:toRGBAString([r,g,b,1])};
+    if (autoUsesOnlyOutline || fov<30) return {style:Style.DESTINATION_OUTLINE, color:toRGBAString([r,g,b,1])};
```

Comment:
This number should come from the constant so that it doesn't break if someone changes that in getAlpha (you probably need to move the constants out of the functions):
```suggestion
    if (autoUsesOnlyOutline || fov<MIN_FOV) return {style:Style.DESTINATION_OUTLINE, color:toRGBAString([r,g,b,1])};
```

Reply:
@robyww Before you merge: I noticed your last commit changed `minFov` in getAlpha to 25, but not in L466, they must be the same. That's why I suggested defining them as `MIN_FOV` constant and using it at these both places to prevent getting them out of sync.

```js
    minFov = 25;
    if (autoUsesOnlyOutline || fov<minFov) return {style:Style.DESTINATION_OUTLINE, color:toRGBAString([r,g,b,1])};
    const alpha= getAlpha(fov, {minFov});
```

Reply:
They don't have to be the same. if FOV < 30 then the `getAlpha()` never gets called. By making minFov less then 30 then it approaches the "fill disappear effect" that you suggested. However, it will never actually gets there, it switches over first. We don't want to get to the "fill disappear" because there will be a FOV that seems buggy.

Reply:
We could accomplish a similar effect by making minFov 30 and minAlpha 5. I might plan with it more at another time.